src/numpy.jl

```diff
@@ -175,14 +175,20 @@ const npy_typestrs = Dict( "b1"=>Bool,
 #########################################################################
 # no-copy conversion of Julia arrays to NumPy arrays.
 
-function PyObject{T<:NPY_TYPES}(a::StridedArray{T})
+# In some cases, an array is already in row-major order (e.g., RGB
+# images) and it is useful to reverse the order of dimensions passed
+# to PyArray_New.  In these cases, set revdims=true.
+
+function PyObject{T<:NPY_TYPES}(a::StridedArray{T}, revdims::Bool=false)
     try
```

Comment:
This should not silently fall back on failure to array2py(a) if revdims=true.  It produces inconsistent behavior.  Is there some way array2py could take revdims as an extra arg as well?